docs/pages/custom-builds/schema.mdx

```diff
@@ -1392,6 +1392,83 @@ build:
   href="https://github.com/expo/eas-build/blob/main/packages/build-tools/src/steps/functions/startIosSimulator.ts"
 />
 
+#### `eas/send_slack_message`
+
+Sends a specified message to a configured Slack webhook URL, which then posts it in the related Slack channel.
+For more information on how to use Slack webhooks check [Slack documentation](https://api.slack.com/messaging/webhooks).
```

Comment:
```suggestion
Sends a specified message to a configured [Slack webhook URL](https://api.slack.com/messaging/webhooks), which then posts it in the related Slack channel.
```

No need to explicitly call out slack's docs here. We can just link it to the keyword in the first sentence.